waspc/src/Wasp/Generator/DbGenerator/Common.hs

```diff
@@ -19,10 +19,10 @@ data DbRootDir
 
 data DbTemplatesDir
 
--- | This file represents the checksum of schema.prisma
--- at the point at which `prisma db migrate-dev` was last run. It is used
--- to help warn the user of instances when they may need to migrate.
-data DbSchemaChecksumOnLastMigrateFile
+-- | This file represents the checksum of schema.prisma at the point
+-- at which we last interacted with the DB to ensure they matched.
+-- It is used to help warn the user of instances when they may need to migrate.
+data DbSchemaChecksumOnLastDbConcurrenceFile
```

Comment:
I'm not sold on the naming of "last concurrence", but mainly the thing I want to capture is we know the schema is the same as the DB because we either:
a) forced it with a migrate, or
b) checked it

Concurrence basically means agreement. Definitely open to other terms that better capture it.

Reply:
Ok, so it was the last time we confirmed that schema is in sync with the db.
Maybe `DbSchemaChecksumOnLastDbSyncFile`? But ok that sounds like some syncing occurred. Hmmmmm not great. Maybe concurrence is ok!

Reply:
Yeah, I had the same initial thought on "sync". Works fine for migrate, but not for checking... we can keep thinking, and if nothing better pops up leave concurrence until something better does.

Reply:
Ok let's go with concurrence then! It is a cool word at least.